app/src/main/java/com/example/bookwormadventuresdeluxe2/EditBooksActivity.java

```diff
@@ -0,0 +1,140 @@
+package com.example.bookwormadventuresdeluxe2;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.bookwormadventuresdeluxe2.Utilities.Status;
+import com.google.zxing.integration.android.IntentIntegrator;
+import com.google.zxing.integration.android.IntentResult;
+
+public class EditBooksActivity extends AppCompatActivity
+{
+    TextView takePhoto;
+    ImageView bookPicture;
+    EditText titleView, authorView, descriptionView, isbnView;
+
+    private static final int CAMERA_REQUEST = 1888;
+    private static final int MY_CAMERA_PERMISSION_CODE = 100;
+
+    // random numbers to identify what the activity will do
+    public static int ADD_BOOK = 50853;
```

Comment:
Is there a better way to do this? What about an enum?

Reply:
Yeah that's a good idea. I can add an enum called IntentMessage? Would that be good even if I'm only using it this one time?

Reply:
I think that might be better, I don't really like the random numbers